puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/wrappers/abort_build.yaml

```diff
@@ -0,0 +1,8 @@
+- wrapper:
+    name: abort_build
```

Comment:
We've some other uses of this in systest_foreman and deploy_deb, did you want to refactor that in at the same time?